os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/common/os/tizenrt/osif_tizenrt.c

```diff
@@ -370,12 +386,12 @@ bool osif_sem_take(void *p_handle, uint32_t wait_ms)
 		ts.tv_sec += wait_ms / 1000;
 		ts.tv_nsec += (wait_ms % 1000) * 1000 * 1000;
 		if (sem_timedwait((sem_t *) p_handle, &ts) != OK) {
-			printf("[osif_sem_take] sema wait 0x%x ms fail\r\n", wait_ms);
+			dbg("sema wait 0x%x ms fail\r\n", wait_ms);

```

Comment:
`wait_ms` is `uint32_t` type. Is it intended to print with `%x` format?

Reply:
It is a incorrect format, done modify.